app/src/main/res/layout/activity_book_details.xml

```diff
@@ -17,8 +17,8 @@
         android:layout_height="wrap_content"
         android:layout_gravity="bottom|end"
         android:layout_marginBottom="16dp"
-        app:fab_addButtonColorNormal="#5f4339"
-        app:fab_addButtonColorPressed="#744F43"
+        app:fab_addButtonColorNormal="#7F00FF"
```

Comment:
You can extract these colors out to the `colors.xml` file. [Android Docs Link](https://developer.android.com/guide/topics/resources/more-resources#Color)